prisma/schema.prisma

```diff
@@ -31,8 +31,9 @@ model User {
   createdAt DateTime  @default(now())
   updatedAt DateTime? @updatedAt()
   deletedAt DateTime?
-  createdBy Int?
-  updatedBy Int?
+  createdBy String?
+  updatedBy String?
+  deletedBy String?
```

Comment:
will this be updated on soft delete? @saroj-rumsan

Reply:
deletedBy?

Reply:
Not in this commit but for the purpose of the audit we have to do it.

Reply:
lets do that too..  tyo ni update garam on softDelete